upup/pkg/fi/cloudup/awstasks/load_balancer.go

```diff
@@ -527,7 +528,34 @@ func (_ *LoadBalancer) RenderAWS(t *awsup.AWSAPITarget, a, e, changes *LoadBalan
 				actualSubnets = append(actualSubnets, fi.StringValue(s.ID))
 			}
 
-			return fmt.Errorf("subnet changes on LoadBalancer not yet implemented: actual=%s -> expected=%s", actualSubnets, expectedSubnets)
+			newSubnetIDs := slice.GetUniqueStrings(actualSubnets, expectedSubnets)
+			oldSubnetIDs := slice.GetUniqueStrings(expectedSubnets, actualSubnets)
+
+			if len(newSubnetIDs) > 0 {
+				request := &elb.AttachLoadBalancerToSubnetsInput{}
+				request.SetLoadBalancerName(loadBalancerName)
+				request.SetSubnets(aws.StringSlice(newSubnetIDs))
+
+				glog.V(2).Infof("Attaching Load Balancer to new subnets")
+
+				_, err := t.Cloud.ELB().AttachLoadBalancerToSubnets(request)
```

Comment:
we can probably wrap into if _, err := <code>; err != nil {

Reply:
Sure, updated.